utils/build/docker/python/uds-flask.Dockerfile

```diff
@@ -2,7 +2,10 @@ FROM datadog/system-tests:flask-poc.base-v3
 
 WORKDIR /app
 
-RUN pip install boto3 kombu
+# this is necessary for the mysqlclient install
```

Comment:
Is it possible to add it in the base image ? it'll speed up the build time

Reply:
done